frontend/src/use-cases/songs/screens/view-songs/components/song-detail/SongDetail.component.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {
+    DigitButton,
+    DigitChip,
+    DigitDesign,
+    DigitLayout,
+    DigitMarkdown,
+    DigitText
+} from "@cthit/react-digit-components";
+import { navEditSong, navHome } from "../../../../../../app/App.routes";
+import SongDetailContainer from "../../../../components/song-detail-container";
+
+const SongDetails = (admin, s, tags, history, text) => {
+    const melody = s.melody ? s.melody : text.Unknown;
+    const author = s.author ? s.author : text.Unknown;
+
+    return {
+        title: s.number + ". " + s.title,
+        renderMain: () => (
+            <SongDetailContainer>
+                <DigitLayout.Row
+                    flexWrap={"wrap"}
+                    justifyContent={"space-between"}
+                >
+                    <div>
+                        <DigitText.Text
+                            bold
+                            text={text.Author + ": " + author}
+                        />
+                        <DigitText.Text text={text.Melody + ": " + melody} />
+                    </div>
```

Comment:
Why use `div`?

Reply:
Just wanted to group them together. Might as well use a Fragment if that is what you are wondering.